### ISSUE TITLE
Footer content update

### ISSUE THREAD

**Issue by emilyville** [department-of-veterans-affairs/vets-website#4439]
Updating the footer to include new things like call center, with some adjustments to accommodate. All the links match pre-existing things. 
![footer-contentchange](https://cloud.githubusercontent.com/assets/1307774/21335254/6c76f660-c612-11e6-9cee-dd809608ebc2.png)

![footer-contentchange-mobile](https://cloud.githubusercontent.com/assets/1307774/21335347/009e85a6-c613-11e6-9a28-85b881ef64dd.png)

[Will update with copy from the images into the issue tomorrow, I need th chat URL still]

@AnitraAppa let me know if you have any edits/thoughts, I believe this is all consistent with the homepage aside from the help desk being added!

The only change in functionality is that the crisis line (now in the header and the footer) should open the modal on desktop like it does on mobile, and remove the down carat from the top.

**Comment by AnitraAppa** [comment]
@emilyville Thanks for flagging! Some thoughts and content recos:
- Use sentence case instead of title case. Easier to read, save title case for biggest header on a page. (Going to do a sentence case scrub across the site - thanks for filing, Gina!)
- Under "Services and Benefits" we have a bunch of links for main pages that prominently show cards that link to the same things under the "Popular on Vets.gov" section. Example: We have both "Heath care" and "Apply for Health Care", "Education Benefits" and "Education Benefits by School". To simplify the footer and avoid redundant looking links, could we just link to the main sections?
- I put "Health care" on the top of the "Resources" section because I believe it's the most used section - is that right?
- I moved "Find Nearby VA locations" under "Services" instead
- Changed "Careers and Employment" to "Employment help"

Took a pass below:

**BENEFITS**
Health care
Disability
Education

**SERVICES**
Nearby VA facilities
Employment help
Refill prescriptions

**CONTACT**
Call the Vets.gov Help Desk
1-855-574-7286 or use chat
Monday‒Friday, 8:00 a.m.‒8:00 p.m. (ET)
Get help from the Veterans Crisis Line

**Comment by dannychapman** [comment]
I really like this! One thought on the horizontal links at the bottom - it might be worth upping the contrast / weight for accessibility.

**Comment by crwallace** [comment]
@emilyville -- I would +1 @dannychapman's comment. Can you either attach the sketch file here or drop it in the core team channel so we can see all of the colors, fonts, font-weights, etc?

**Comment by emilyville** [comment]
@crwallace @dannychapman I was trying not to introduce anything new with this, so we shouldn't need a sketch file. I can make it full white when I incorporate Anitra's copy changes. (will update today)

**Comment by emilyville** [comment]
Thanks everyone! Here's the final versions. 

**Design notes:** I combined the last two columns so it's still 4 wide, but the last two are combined to be the help desk and take up 50% of the width. 

**Content notes:** @AnitraAppa - Thank you for looking this over! I kept this consistent with the homepage sections and options. You're totally right about sentence case and ordering, so I fixed those below. All the other changes we should address if we want to also make those section changes on the home page as well, but I think we should treat as a separate work item. Aside from all caps which we're no longer planning on using. 

![footer-contentchange-mobile](https://cloud.githubusercontent.com/assets/1307774/21623960/b446ae92-d1b9-11e6-832e-28332a70b897.png)

![footer-contentchange](https://cloud.githubusercontent.com/assets/1307774/21624002/df914526-d1b9-11e6-80b1-15b33ed69ee4.png)

**Services and Benefits**
Disability Benefits
Education Benefits
Careers and Employment
Health Care

**Popular on Vets.gov**
Apply for health care
Apply for education benefits
Check disability claim status
Education benefits by school
Refill prescriptions

_Supporting links at the bottom - with two additional links_
Vets.gov Playbook | Message from the Secretary 
VA.gov Inspector General | Accessibility | Privacy | USA.gov

Vets.gov Playbook --> https://www.vets.gov/playbook/
Message from the Secretary --> https://www.vets.gov/2016/11/11/help-us-improve-vets-gov
